core/src/test/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorTests.java

```diff
@@ -65,6 +67,11 @@ public void testEqualsAndHashcode() throws IOException {
         }
     }
 
+    public void testLevensteinDeprecation() {
+        assertThat(DirectCandidateGeneratorBuilder.resolveDistance("levenstein"), equalTo(new LevensteinDistance()));
```

Comment:
I might be missing something but I do not see a test that `resolveDistance("levenshtein")` does the right thing and does not produce a deprecation warning?

Reply:
Good catch!